tests/test_inventory.sh

```diff
@@ -0,0 +1,97 @@
+#! /bin/sh
+
+test_dir="$(dirname $0)"
+tmp_dir="$(mktemp -d)"
+state_dir="$(mktemp -d)"
+inv_script="${test_dir}/../src/mender-inventory-mender-configure"
+
+# Override variables inside the inventory script.
+export TEST_CONFIG="${state_dir}/device-config.json"
+export TEST_HTTP_LOG="${tmp_dir}/http.log"
+
+oneTimeSetUp() {
+    "${test_dir}/helpers/mender-configure-server.py" &
+    TEST_SERVER_PID=$!
+    sleep 1
+}
+
+oneTimeTearDown() {
+    # For some reason this sometimes returns failure, but the process *is*
+    # killed.
+    kill $TEST_SERVER_PID || true
+}
+
+setUp() {
+    OLD_PATH="$PATH"
+    export PATH="${test_dir}/helpers:$PATH"
```

Comment:
The PATH manipulation here (and the corresponding line in `tearDown`) could be moved to `oneTimeSetUp` (and `oneTimeTearDown`)

Reply:
I thought about that too, but I decided to put it here in case any test would do any local modifications to `PATH`. They don't, at the moment, but I thought it would be better keep every test completely isolated, also for the future.